packages/docusaurus-theme-classic/src/index.ts

```diff
@@ -176,7 +176,7 @@ export default function themeClassic(
 
     configureWebpack() {
       const prismLanguages = additionalLanguages
-        .map((lang) => `prism-${lang}`)
+        .map((lang) => `prism-${lang.toLowerCase()}`)
```

Comment:
This could be better handled in the config validation, you can use Joi to normalize/lowercase the provided user input values so that everything else in the system see lowercase values (including `prism-include-languages.ts`, it shouldn't be needed to lowercase there too anymore)